src/main/kotlin/app/revanced/patcher/fingerprint/method/impl/MethodFingerprint.kt

```diff
@@ -19,6 +19,7 @@ import java.util.*
 private typealias StringMatch = MethodFingerprintResult.MethodFingerprintScanResult.StringsScanResult.StringMatch
 private typealias StringsScanResult = MethodFingerprintResult.MethodFingerprintScanResult.StringsScanResult
 private typealias MethodClassPair = Pair<Method, ClassDef>
+private typealias CustomFingerprint = ((methodDef: Method, classDef: ClassDef) -> Boolean)
```

Comment:
Why is an alias present for this again?

Reply:
Because I think it makes the code more readable, and a typealias allows us to easily re-use it across the codebase.

Reply:
The alias `CustomFingerprint` is only used in this class & it should be called differently.

Reply:
Yes, but in my other PR over at revanced-patcher I added the following:

https://github.com/revanced/revanced-patches/pull/2391/files#diff-22fcba02af7ed74fb20b41d162ed1daf9d95387c1932e4ee691f45b0e13b0f06